### ISSUE TITLE
Is this project from the paper [SplicePort—An interactive splice-site analysis tool]?

### ISSUE THREAD

**Issue by ghost** [MountLab/SplicePort-alpha#1]
Hi, I want to know if you implement this algorithm from the paper [SplicePort—An interactive splice-site analysis tool] ?

**Comment by MountLab** [comment]
Yes. We have created a command-line version of the same software in Java. We also revised the feature tables to be (somewhat) human-readable. The output should be identical.

**Comment by ghost** [comment]
> Yes. We have created a command-line version of the same software in Java. We also revised the feature tables to be (somewhat) human-readable. The output should be identical.

Thank you!
